block-editor/text-decoration-color/src/index.js

```diff
@@ -0,0 +1,174 @@
+/**
+ * Flex Grow Control
```

Comment:
Can this be changed to Text Decoration Color?

Reply:
Yes, updated!